### PR TITLE
fix(i10n): update

### PR DESCRIPTION
**Pull request by vomikan**
Polish

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1985)

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d412bb3743cfa1ee6848866b0dfcf5618832ff57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 873a9f83dcc52998b2809d97b31528020b52a947
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 8976338dc07f5ffa39f81be367598fbe64196e82
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 01ad7a05cc1a36bd244319020ec2ad8262d0d5f7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/pl_PL.po [hunk 1]
```diff
@@ -2,30 +2,33 @@ msgid ""
...
 "Language-Team: Polish\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: crowdin.com\n"
-"X-Poedit-SourceCharset: UTF-8\n"
+"X-Poedit-Basepath: ../src\n"
+"X-Poedit-KeywordsList: _;wxTRANSLATE\n"
+"X-Poedit-SearchPath-0: .\n"
+"X-Poedit-SearchPathExcluded-0: db/DB_Upgrade.h\n"
```
Comment (slodki) [inline po/pl_PL.po]: no need to add this outdated Poedit settings

File: po/pl_PL.po [hunk 2]
```diff
@@ -1366,7 +1369,7 @@ msgstr "Pokaż podsumowanie kategorii budżetu"
 #: budgetingpanel.cpp:184 reports/budget.cpp:84
 #, c-format
 msgid "Financial Year: %s - %li"
-msgstr "rok finansowy: %s - %li"
+msgstr "Rok finansowy:%s -%li"
```
Comment (slodki) [inline po/pl_PL.po]: old is better

File: po/pl_PL.po [hunk 3]
```diff
@@ -2029,7 +2032,7 @@ msgid "Other Expenses"
 msgstr "Inne wydatki"
 
 #: db/Table_Category.h:120 model/Model_Billsdeposits.cpp:33
-#: model/Model_Checking.cpp:32 transdialog.cpp:335
```
Comment (slodki) [inline po/pl_PL.po]: hmmmm?

File: po/pl_PL.po [hunk 4]
```diff
@@ -6611,14 +6614,14 @@ msgstr "Wybierz źródło aktualizacji"
 msgid "Select preferred version:"
 msgstr "Wybierz preferowaną wersję:"
 
+#: optionsettingsnet.cpp:226
+msgid "WebApp connection tested successfully!"
+msgstr ""
```
Comment (slodki) [inline po/pl_PL.po]: why empty? there is translation removed below...

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d412bb3743cfa1ee6848866b0dfcf5618832ff57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 873a9f83dcc52998b2809d97b31528020b52a947
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 8976338dc07f5ffa39f81be367598fbe64196e82
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 01ad7a05cc1a36bd244319020ec2ad8262d0d5f7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: a0a58f9441b30f130e0982d24f20f9332a5c87fc
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/pl_PL.po (hunk 1 above)
Comment (vomikan) [reply po/pl_PL.po]: I've updates mmex.pot file from sources by poedit tool. I don't know other and working for me way.

**Comment by vomikan** [comment]
>           Can we approve translations directly at Crowdin? I have no access to approve - only to vote.
> It's unnecessary to translate at Crowdin than check translation few minutes later at GitHub.
> Some changes in this PR needs checking reasons for strange updates...

image

Seems this behavior regarding your role. I've changed it to Manager (Please do not update html files sources).

File: po/pl_PL.po (hunk 1 above)
Comment (slodki) [reply po/pl_PL.po]: Do you want a Windows version of update-po-files.sh? I think Poedit will be not enought to generate complete .pot file from all sources (see https://github.com/slodki/moneymanagerex/blob/29b05f50f658ab1e092c69a5fab9344f1a010fe0/util/update-po-files.sh#L22)

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d412bb3743cfa1ee6848866b0dfcf5618832ff57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 873a9f83dcc52998b2809d97b31528020b52a947
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 8976338dc07f5ffa39f81be367598fbe64196e82
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 01ad7a05cc1a36bd244319020ec2ad8262d0d5f7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: a0a58f9441b30f130e0982d24f20f9332a5c87fc
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: bd433d4dc2ae088b35390e1b651bf78c88b52760
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/pl_PL.po (hunk 3 above)
Comment (slodki) [reply po/pl_PL.po]: But it should be included. There is `-kwxGetTranslation` option in update-po-files.sh script.

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d412bb3743cfa1ee6848866b0dfcf5618832ff57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 873a9f83dcc52998b2809d97b31528020b52a947
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 8976338dc07f5ffa39f81be367598fbe64196e82
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 01ad7a05cc1a36bd244319020ec2ad8262d0d5f7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: a0a58f9441b30f130e0982d24f20f9332a5c87fc
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: bd433d4dc2ae088b35390e1b651bf78c88b52760
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/pl_PL.po (hunk 1 above)
Comment (vomikan) [reply po/pl_PL.po]: Poedit is working excellent. Last update of pot has done by Poedit.
And it has interface with Crowdin. 
image

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d412bb3743cfa1ee6848866b0dfcf5618832ff57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 873a9f83dcc52998b2809d97b31528020b52a947
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 8976338dc07f5ffa39f81be367598fbe64196e82
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 01ad7a05cc1a36bd244319020ec2ad8262d0d5f7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: a0a58f9441b30f130e0982d24f20f9332a5c87fc
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: bd433d4dc2ae088b35390e1b651bf78c88b52760
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: 993b866a63ce45efd35c69322db51e2fa838dbee
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Pull request merged**